src/coreclr/scripts/superpmi.py

```diff
@@ -327,6 +337,21 @@
 asm_diff_parser.add_argument("-retainOnlyTopFiles", action="store_true", help="Retain only top .dasm files with largest improvements or regressions and delete remaining files.")
 asm_diff_parser.add_argument("--diff_with_release", action="store_true", help="Specify if this is asmdiff using release binaries.")
 
+# subparser for throughput
+throughput_parser = subparsers.add_parser("tpdiff",
+                                          description=throughput_description, parents=[core_root_parser, target_parser, superpmi_common_parser, replay_common_parser],
+                                          # Allow overriding build_type with new default
+                                          conflict_handler='resolve')
+
+# Override build type
+throughput_parser.add_argument("-build_type", default="Release", help=throughput_build_type_help)
+throughput_parser.add_argument("-base_jit_path", help="Path to baseline clrjit. Defaults to release baseline JIT from rolling build, by computing baseline git hash.")
+throughput_parser.add_argument("-diff_jit_path", help="Path to diff clrjit. Defaults to release Core_Root JIT.")
+throughput_parser.add_argument("-git_hash", help="Use this git hash as the current hash for use to find a baseline JIT. Defaults to current git hash of source tree.")
+throughput_parser.add_argument("-base_git_hash", help="Use this git hash as the baseline JIT hash. Default: search for the baseline hash.")
+throughput_parser.add_argument("-base_jit_option", action="append", help="Option to pass to the baseline JIT. Format is key=value, where key is the option name without leading COMPlus_...")
+throughput_parser.add_argument("-diff_jit_option", action="append", help="Option to pass to the diff JIT. Format is key=value, where key is the option name without leading COMPlus_...")
```

Comment:
Looks like you could add a "diff_parser" parent for shared things between asm_diff_parser and throughput_parser, but I'm ok either way.

Reply:
Done.